apps/protocol-frontend/src/hooks/useContributionList.ts

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react';
+import { useUser } from '../contexts/UserContext';
+import { useQuery, useInfiniteQuery } from '@tanstack/react-query';
+import { UIContribution } from '@govrn/ui-types';
+import { ListContributionsQueryVariables } from '@govrn/protocol-client';
+
+export const useContributionList = (args?: ListContributionsQueryVariables) => {
+  const { govrnProtocol: govrn } = useUser();
+  const { isLoading, isFetching, isError, error, data } = useQuery(
+    ['contributionList', args],
+    async (): Promise<UIContribution[]> => {
+      const data = await govrn.contribution.list(args || {});
```

Comment:
This is a much more streamlined check for args existing. I think I used a slightly more verbose way in the `useDaosList` -- I'll double check and update to match this approach